packages/client-beta/src/apps/Soundboard/Soundboard.tsx

```diff
@@ -25,9 +28,34 @@ function Soundboard() {
   createEffect(() => {
     fetch('/api/x-sounds')
       .then((res) => res.json())
-      .then((res) => setXsounds(res.sounds));
+      .then((res) => {
+        setXsounds(res.sounds);
+        setFilteredXSounds(res.sounds);
+      });
+
+    fetch('/api/x-sounds/tags')
+      .then((res) => res.json())
+      .then((res) => setTags(res.tags));
   });
 
+
+  const onSearchChange = (e: Event) => {
+    const phrase = (e.target as HTMLInputElement).value;
+    const name = (e.target as HTMLInputElement).name;
+
+    if(name === 'phrase') {
+      (document.getElementsByName('tags')[0] as HTMLInputElement).value = '';
```

Comment:
It's because getElementsByName returns Element type, and Element type doesnt have a "value".
<img width="389" alt="image" src="https://github.com/user-attachments/assets/506d4486-f1e2-4a12-aaf4-e7738cb84f18">
Maybe a better way to catch this would be to give these inputs an id. Then we don't have to do [0]. Or maybe querySelector('input[name=tags'). Didn't think about it :D I'd still have to do that casting, because each returns an Element

Reply:
I don't have a problem with all of that, I was just surprised that there is no way to get the native HTML element using Solid and we have to use document querying.

Like React has `ref={this.myElement}` or something like that and Solid does not?